### ISSUE TITLE
Remove VACOLS login

### ISSUE THREAD

**Issue by shanear** [department-of-veterans-affairs/caseflow-certification#311]
Instead map CSS Station IDs to RO ids, and use that to know the user's RO.

Depends on #310

Current plan:
- [x] Create a mapping of Station IDs to RO ids
- [x] Show RO id using the mapping instead of the VACOLS id
- [ ] Remove VACOLS login code

**Comment by shanear** [comment]
@KHarshawat @gmorgana-va do either of yall know where to find this mapping?

**Comment by KHarshawat** [comment]
https://github.com/department-of-veterans-affairs/appeals-pm/blob/master/other%20systems/VACOLS/ro_list.xls

**Comment by KHarshawat** [comment]
Emailed Chris Denno to see if I could get mappings for the remaining ROs with weird edge cases like those that end in something like RACC or COWAC, Pension Center, or Education Center.

**Comment by shanear** [comment]
@KHarshawat  If it's helpful, we could run some reports on which of these ROs have ever had a ceritfication done with caseflow. That way we'd know which ones to account for

**Comment by KHarshawat** [comment]
Sure that'd be super helpful! Chris mentioned that may of these are not even used any more.

**Comment by KHarshawat** [comment]
@shanear updated mapping of RO> Station ID is here: https://github.com/department-of-veterans-affairs/appeals-pm/blob/master/bva%20documentation/ro_list.xls

This is currently on hold until we can find an answer to: https://github.com/department-of-veterans-affairs/appeals-pm/issues/679

**Comment by KHarshawat** [comment]
We've determined that we don't need to restrict usage by RO (department-of-veterans-affairs/appeals-pm#679) so I'm removing the on-hold label and moving this into the sprint backlog.

**Comment by shanear** [comment]
This is blocked until we start using CSS on 9/26

**Comment by shanear** [comment]
After a conversation with Jed, it looks like this one will be tricker than we thought:

- 10 of the 58 Station IDs have multiple regional offices associated to them, and according to Jed, it's important for them to be correct, so we would need to force users in those RO's to disambiguate.
- The regional offices that share the same station ID are always in the same city and state. We've received feedback that sometimes users want to certify appeals from other RO id's in their city and it's annoying to log in and log out to do this.

## Proposed solution
- If there is a 1 to 1 Station ID / RO ID mapping, then skip the VACOLS login
- If not, keep the VACOLS login for those users

- For certification, the only reason getting the RO correct matters is the RO id entered into the Form 8, it may be ok to just use the RO already listed on the appeal for that field, and punt on the disambiguation
- Then for future caseflow features, speak with DRO's from the regions where they have multiple RO's and see if there's an easier way for them to disambiguate which RO they are acting under (like an RO picker?)

What do you think @KHarshawat?

**Comment by shanear** [comment]
postponing this for #325, since doing that will make this considerably easier
